### PR TITLE
#AV-97946 Update travis python version to python 3.7

### PR DESCRIPTION
**Pull request by shounak-opcito**
Resolved ansible-lint issues

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +16,7 @@ services:
   - docker
 
 install:
-  - pip install molecule docker pluggy==0.13.1 yamllint ansible-lint docker
+  - pip install molecule docker pluggy==0.13.1 yamllint ansible-lint docker molecule-docker
```
Comment (sabandi) [inline .travis.yml:19]: Why do we need `molecule-docker`

File: .travis.yml [hunk 2]
```diff
@@ -28,8 +30,8 @@ notifications:
...
   email:
     recipients:
-      - grastogi@avinetworks.com
-      - chaitanya.deshpande@avinetworks.com
-      - shrikant.patil@avinetworks.com
-      - rohan.suryawanshi@avinetworks.com
+#      - grastogi@avinetworks.com
+#      - chaitanya.deshpande@avinetworks.com
+#      - shrikant.patil@avinetworks.com
+      - shounak.joshi@avinetworks.com
```
Comment (sabandi) [inline .travis.yml:36]: let's start using vmware emails.

File: molecule/default/molecule.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ driver:
 lint: |
   set -e
   yamllint tasks -c molecule/default/.yamllint
-  ansible-lint molecule/default/converge.yml
+  ansible-lint molecule/default/converge.yml -x '106'
```
Comment (sabandi) [inline molecule/default/molecule.yml:9]: I would suggest using a config file.

File: tasks/cloud.yml [hunk 4]
```diff
@@ -87,17 +87,17 @@
...
+    password: "{{ avi_password  | default(omit) }}"
+    tenant: "{{ item.tenant | default(omit) }}"
     state: "{{ avi_role_state_mode }}"
-    api_version: "{{ api_version | default(omit)}}"
-    avi_credentials: "{{ avi_credentials | default(omit)}}"
-    api_context: "{{avi_api_context | default(omit)}}"
-    name: "{{item.name}}"
+    api_version: "{{ api_version | default(omit) }}"
+    avi_credentials: "{{ avi_credentials | default(omit) }}"
+    api_context: "{{ avi_api_context | default(omit) }}"
```
Comment (sabandi) [inline tasks/cloud.yml:99]: Can we also modify the script which creates these assets to have this spaces.

File: .travis.yml (hunk 1 above)
Comment (shounak-opcito) [reply .travis.yml:19]: With the new version of the molecule, we need to have this dependency.

File: .travis.yml (hunk 2 above)
Comment (shounak-opcito) [reply .travis.yml]: Updated the emails

File: molecule/default/molecule.yml (hunk 3 above)
Comment (shounak-opcito) [reply molecule/default/molecule.yml]: Updated. added file `.ansible-lint`

**Review by sabandi — APPROVED** [review]

**Pull request merged**